README.md

```diff
@@ -1,64 +1,129 @@
 # Amalthea - A Kubernetes operator for Jupyter servers
 
-This project defines a `JupyterServer` custom resoure (CRD) for Kubernetes and
-implements a Kubernetes operator which controls the lifecycle of custom
-JupyterServer objects.
+This project defines a custom `JupyterServer`
+[resource](https://github.com/SwissDataScienceCenter/amalthea/tree/main/helm-chart/amalthea/templates/crd.yaml)
+for Kubernetes and implements a Kubernetes operator which controls the lifecycle
+of custom `JupyterServer` objects.
+
+## Example
+
+Once Amalthea is installed in a cluster through the helm chart, deploying a
+jupyter server for a user `Jane Doe` with email `jane.doe@example.com` is easy
+as applying the following YAML file to the cluster:
+
+```yaml
+apiVersion: amalthea.dev/v1alpha1
+kind: JupyterServer
+metadata:
+  name: janes-spark-session
+  namespace: datascience-workloads
+spec:
+  jupyterServer:
+    image: jupyter/all-spark-notebook:latest
+  routing:
+    host: jane.datascience.example.com
+    path: /spark-session
+    tls:
+      enabled: true
+      secretName: example-com-wildcard-tls
+  auth:
+    oidc:
+      enabled: true
+      issuerUrl: https://auth.example.com
+      clientId: jupyter-servers
+      clientSecret: 5912adbd5f946edd4bd783aa168f21810a1ae6181311e3c35346bebe679b4482
+      authorizedEmails:
+        - jane.doe@example.com
+    token: ""
+```
+
+For the full configuration options check out the
```

Comment:
it's customary to list the configuration options in a table...

Reply:
I'm still looking for a a tool which parses a crd into a table... :) 
But point taken 👍